benchmarks/query_benchmarks/query_multifield/benchmark.py

```diff
@@ -0,0 +1,18 @@
+from ...utils import bench_setup
+from .models import MultiField
+
+
+class QueryMultiField:
+    def setup(self):
+        bench_setup(migrate=True)
+        for i in range(0, 3000):
+            kwargs = {}
+            for j in range(1, 11):
+                kwargs["field%s" % j] = "foobar_%s_%s" % (i, j)
```

Comment:
Could we rewite as a comprehension? Is this equivilent? 

```suggestion
            kwargs = {f"field%s" : f"foobar_{i}_{j}" for j in range(1,11)}
```